🐞 - Failed build Next app after updating to 2.0.0

### Which package(s) are the source of the bug?

@maskito/react

### Description

After updating to version 2.0.0 I get an error
```
 ⨯ ./node_modules/@maskito/react/index.esm.js:1:0
Module not found: Can't resolve 'core-js/modules/web.dom-collections.iterator.js'
```
![image](https://github.com/taiga-family/maskito/assets/15365104/f373b566-ac06-4725-a884-059016301597)


### Maskito version

2.0.0

### Which browsers have you used?

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

--- a/projects/react/.babelrc
+++ b/projects/react/.babelrc
@@ -3,8 +3,7 @@
         [
             "@nx/react/babel",
             {
-                "runtime": "automatic",
-                "useBuiltIns": "usage"
+                "runtime": "automatic"
             }
         ]
     ],

--- a/projects/react/project.json
+++ b/projects/react/project.json
@@ -15,6 +15,7 @@
                 "entryFile": "projects/react/src/index.ts",
                 "external": "all",
                 "rollupConfig": "@nx/react/plugins/bundle-rollup",
+                "compiler": "babel",
                 "assets": [
                     {
                         "glob": "projects/react/README.md",

